draft-irtf-panrg-path-properties.md

```diff
@@ -217,7 +219,13 @@ Finally, a NAT that actively modifies IP and TCP/UDP headers based on their cont
 Note that according to this definition, a node that modifies packets in accordance with the hosts, such as a transparent HTTP proxy, as defined in {{RFC2616}}, and a node listening and reacting to implicit or explicit signals, see {{RFC8558}}, are not considered transparent.
 
 Administrative Domain:
-: The administrative domain, e.g., the IGP area, AS, or Service provider network to which a path element belongs.
+: An individual or an organization that owns a path element (or several path elements).
+Examples of administrative domains are an IGP area, an AS, or a service provider network.
+
+Routing Domain Identifier:
+: An identifier indicating the routing domain of a path element.
+All path elements with the same routing domain identifier are in the same administrative domain and use a common routing protocol to communicate with each other.
```

Comment:
I think here it would be fine to say the following, to make it more of a definition (i.e., these are necessary requirements for path elements to be in the same routing domain, instead of merely statements that are true for some path elements in the same routing domain).
And I think we can still put it like this, even if we keep this an "identifier".

```suggestion
Path elements in the same routing domain are in the same administrative domain and use a common routing protocol to communicate with each other.
```